example/src/App.js

```diff
@@ -15,14 +15,14 @@ const renderMeetingActions = () => {
         </ListItemIcon>
       </ListItem>
       <ListItem>
-        XD
+        Sample Meeting Action
       </ListItem>
     </>
   )
 }
 
 const App = () => {
-  return <MeetingOrganizer userData={ { userType: 'job_seeker', uuid: '264665a1-53eb-40fc-b263-697b0bf88a79' } } />
+  return <MeetingOrganizer userData={ { userType: 'job_seeker', uuid: '8380a212-3e29-430b-9d13-a752036ba6c4' } } />
```

Comment:
To jest do exampla testowo żeby coś się pojawiało?

Reply:
To ogólnie działa tak, że komponent może przyjąć trzy typy userData: host, job_seeker i coś innego. One wyświetlają odpowiednie kalendarze.

Reply:
Ale tak, jest to jakieś losowe uuid z backu lokalnie postawionego, więc nic się nie dzieje, że to w examplu jest.